docs/fundamentals/failed-events/index.md

```diff
@@ -39,7 +39,7 @@ Other failures generally fall into 3 categories:
 
 * **Bots or malicious activity**. Bots, vulnerability scans, and so on, can send completely invalid events to the Collector. The format might be wrong, or the payload size might be extraordinarily large.
 
-* **Pipeline misconfiguration**. For example, a loader could be reading from the wrong stream (with events in the wrong format). This is quite rare, especially for Snowplow BDP, where all relevant pipeline configuration is automatic.
+* **Pipeline misconfiguration**. For example, a loader could be reading from the wrong stream (with events in the wrong format). This is quite rare, unless you're self-hosting, as all relevant pipeline configuration is automatic.
```

Comment:
```suggestion
* **Pipeline misconfiguration**. For example, a loader could be reading from the wrong stream (with events in the wrong format). This is quite rare, unless you're self-hosting Snowplow, as all relevant pipeline configuration is automatic.
```

To avoid ambiguity w.r.t. different things a customer might be self-hosting (like trackers)